src/test/java/io/vertx/core/spi/cluster/ClusterManagerDelegate.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2011-2020 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.core.spi.cluster;
+
+import io.vertx.core.Future;
+import io.vertx.core.impl.VertxInternal;
+import io.vertx.core.shareddata.AsyncMap;
+import io.vertx.core.shareddata.Counter;
+import io.vertx.core.shareddata.Lock;
+
+import java.util.List;
+import java.util.Map;
+
+public class ClusterManagerDelegate implements ClusterManager {
```

Comment:
this class is unused and does not do much beside implementing the delegation pattern

Reply:
It's a leftover from previous commits: the delegate was used in tests to override specific methods (e.g. to make sure a registration never completes). I'll remove it.